backend/local_deployment/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.2'
+
+services:
+  db:
+    build: ./postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - 5432:5432
+  service:
+    image: trackyourbed
+    restart: always
+    network_mode: host
+    environment:
+      DB_HOST: 127.0.0.1
```

Comment:
Would the container name `db` work here? In docker, it should resolve to the internal docker ip for the database server.

Reply:
Yes that works, so then we can get rid of the network_mode as well, thank you!